client/src/policyDetail/RelatedPolicyDetails.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const RelatedPolicyDetails = ({ policy, id }) => {
+  return (
+    <p data-testid={'relatedPolicy'}>
+      <a href={`./${id}`}>{policy}</a>
+    </p>
+  )
+}
+RelatedPolicyDetails.propTypes = {
+  policy: PropTypes.string,
+  id: PropTypes.string,
```

Comment:
id: PropTypes.number. will fix the string/number problem (see other comment).
Not sure if the fix should be here or above, but I think here.